examples/README.md

```diff
@@ -0,0 +1,125 @@
+
+1. Navigate to the folder where the cloned repository is.
+2. Inside the folder, open up the Julia REPL with the command `julia --project=.`. This will signal Julia to utilize the environment inside the folder, described by the `Project.toml`  and the `Manifest.toml` files included with the package.
+3. on the Julia REPL, press the `]` key on the console. The cursor will change to `(Resonance) pkg>`, which is called Package Mode. Double-check that the Package name (Resonance) is displayed within parentheses. if you see the Julia version (ex: `(@v1.7) pkg>`, `(@v1.8) pkg>`, etc.), you need to backtrack and make sure to start the process on the root of the package directory with the adequate arguments.
+4. On Pkg mode, type `instantiate` and hit enter. This command will install the compehensive list of dependencies on your `Project.toml` file.
+5. Wait for precompilation and take note of any warnings, following the insctructions on screen to deal with them. After precompilation, you will see the command line input again.
+6. Test that the `Resonance` package can be loaded: hit `backspace` to go back to the julia REPL and type `using Resonance`. If you do not get an error message withing a few seconds, you are ready to use the example scripts.
+
+### (Optional) Download some of our pretrained models
+
+if you have data that matches the format and acquisition constraints in the RESONANCE package publication, you may want to use our own models while predictiong your own data. In order to do so, you have to obtain the model binaries separetely, since they are not included on this repository. In order to do that, follow the instructions:
+
+1. Navigate to the Google Drive folder where the models are located: https://drive.google.com/drive/u/2/folders/1Av3TyvfiOIy5PbTMgsZiB6MRKNmzdWCw
+2. If you cannot access the files, email [Kevin Bonhan](mailto:kbonham@wellesley.edu) or [Guilherme Bottino](mailto:gz101@wellesley.edu) regarding granting you access. You may also use the "request access" functionality on Google Drive.
+3. Download the models and extract them to the `models/` folder inside the package root directory.
+
+## Running the example scripts
+
+### Training your own models
+
+The first example script is `examples/train_univariate_randomforest.jl`. This script takes five **mandatory** arguments:
+
+`julia examples/train_univariate_randomforest.jl {-c | -r} input_csv input_cols output_col output_filename`
+
+1. The first argument tells the script to run in either Regression or Classification mode.
+
+2. The second argument is a community profile (with subjects/samples as rows, and features/predictors as columns) on the CSV format. The script expects a header for the CSV amd may fail if a header line is not provided. Please refer to the provided `examples/example_taxonomic_profile.csv` for formatting.
```

Comment:
sembr please, I'm going to stop adding them

Reply:
added sembr